pkg/platform/model/buildplanner.go

```diff
@@ -287,13 +287,13 @@ func (bp *BuildPlanner) StageCommit(params StageCommitParams) (strfmt.UUID, erro
 	}
 
 	if resp.Error != nil {
-		return "", errs.New("Failed to stage commit: " + resp.Error.Message)
+		return "", locale.NewInputError("Failed to stage commit, API returned message: {{.V0}}", resp.Error.Message)
```

Comment:
```suggestion
		return "", locale.NewError("Failed to stage commit, API returned message: {{.V0}}", resp.Error.Message)
```

Unless we know for certain that this is an input error?

Reply:
I recall we didn't want to flood rollbar with buildplanner errors and you followed up with PB to ensure they are being logged by them. That was my reasoning for changing it but I'm happy to make it a regular error if that's not the case.

Reply:
That's right, but this was handled in one place through the buildplan error type, which I assumed is wrapped around this. I don't want to have this special handling of errors implemented in different ways throughout the package.